rust/platform/platform.bzl

```diff
@@ -0,0 +1,42 @@
+load(":triple_mappings.bzl", "triple_to_constraint_set")
+
+# All T1 Platforms should be supported
+#
+# TODO: Review: How to handle stdlib variants? There doesn't appear to be a way to distinguish between them based on the constraint values available in @bazel_tools//platforms
```

Comment:
I can't offer anything here, perhaps there is something in the CROSSTOOL that would help us figure out if musl is setup? I would be surprised if there is a simple answer to this.

Reply:
I've changed this from a "review" todo to a simple statement, so, i'm punting on it.